upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -338,6 +338,96 @@ There you can find links to upgrade notes for other versions too.
           +    <i class="list-menu__item__control svg svg-arrow js-category-collapse-control" data-url="{{ url('front_category_branch', { parentCategoryId: categoryWithLazyLoadedVisibleChildren.category.id }) }}"></i>
             {% endif %}
           ```
+- apply these changes to add support for naming uploaded files([#1547](https://github.com/shopsys/shopsys/pull/1547))
+    - update your `composer.json`
+        ```diff
+            "ext-curl": "*",
+        +   "ext-fileinfo": "*",
+            "ext-gd": "*",
+        ```
+    - update your `docker/php-fpm/Dockerfile`
+        ```diff
+            RUN docker-php-ext-install \
+                bcmath \
+        +       fileinfo \
+                gd \
+        ```
+    - add this route to the end of your `config/routes/shopsys_front.yml`
+    ```diff
```

Comment:
You have wrong indenting here and bellow

Reply:
Fixed